### ISSUE TITLE
MacOS: Results Grid Not Displaying Tooltip in Full Screen Mode

### ISSUE THREAD

**Issue by Xylandor** [dbeaver/dbeaver#12721]
#### System information: 
- macOS Catalina - version 10.15.6
- DBeaver version 21.1.0.202105300349
- Additional extensions: None

#### Connection specification:
- Database name and version: MySQL 5.7
- Driver name: MySQL 5
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
In the query results pane, if a column contains data that is larger than what is shown, hovering the cursor over the column displays a tooltip showing more of the content. Ever since the release of DBeaver 21.0.5, the tooltip display will no longer show when the DBeaver window is in Mac's fullscreen mode. When I take it out of fullscreen mode, the tooltip will appear as expected. It worked perfectly fine in fullscreen, until I upgraded to DBeaver 21.0.5. I waited to see if 21.1 would fix the issue, but it did not. 

#### Steps to reproduce, if exist:
1. On the Mac, set DBeaver to fullscreen mode.
2. Run a query where a column will contain many characters, such as
   `SELECT 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam non lacus eu mi interdum ornare ac a neque.' AS Test`
3. In the results pane, reduce the column width to where only some of the text in the column is visible. 
4. Hover the cursor over the column content. The tooltip is supposed to be displayed to show you the rest of the data, but it does not.
5. Now exit fullscreen mode and hover back over the column. After a short delay, the tooltip now shows as expected.
6. Enter fullscreen mode again and the tooltip will not show when hovering over the content.  

Here's a screenshot of the expected behavior that does not happen when in fullscreen mode:
![dbeaver_tooltip_issue](https://user-images.githubusercontent.com/24617679/120651930-e8f32080-c444-11eb-97cb-e18114ae4bf0.png)

#### Include any warning/errors/backtraces from the logs
No errors or messages of any kind are produced/presented. From the time I connected to my database to running through the reproductive steps, below are the only log entries found in the dbeaver-debug.log file. 

2021-06-03 07:40:15.433 - Connect with 'ICP' (mysql5-175c4c030e9-168289efb33415b9)
> Start DBeaver MySQL Support [org.jkiss.dbeaver.ext.mysql 2.1.143.202105300349]
2021-06-03 07:40:15.466 - Execution context opened (ICP; Main; 0)
> Start DBeaver GIS Support [org.jkiss.dbeaver.data.gis 1.0.82.202105300349]
2021-06-03 07:40:17.076 - Connected (mysql5-175c4c030e9-168289efb33415b9, MySQL Connector Java [mysql-connector-java-5.1.44 ( Revision: b3cda4f864902ffdde495b9df93937c3e20009be )])
> Start DBeaver MySQL UI Support [org.jkiss.dbeaver.ext.mysql.ui 1.0.50.202105300349]
> Start DBeaver Tasks - General UI [org.jkiss.dbeaver.tasks.ui 1.0.36.202105300349]
> Start DBeaver UI Editors - Connections [org.jkiss.dbeaver.ui.editors.connection 1.0.49.202105300349]
> Start DBeaver SSH tunnels UI [org.jkiss.dbeaver.net.ssh.ui 1.0.79.202105300349]
> Start DBeaver SSH tunnels [org.jkiss.dbeaver.net.ssh 1.0.79.202105300349]
2021-06-03 07:40:21.189 - Service 'org.jkiss.dbeaver.runtime.ui.UIServiceSecurity' not found
2021-06-03 07:41:39.747 - Service 'org.jkiss.dbeaver.runtime.ui.UIServiceSecurity' not found
> Start Team Support Core [org.eclipse.team.core 3.8.1100.v20200806-0621]
2021-06-03 07:51:37.925 - Execution context opened (ICP; SQLEditor <Console>; 1)
> Start Core Compare Support [org.eclipse.compare.core 3.6.1000.v20201020-1107]
> Start Console [org.eclipse.ui.console 3.10.100.v20201211-1511]
> Start Debug Core [org.eclipse.debug.core 3.18.0.v20210222-1101]
> Start GEF (MVC) [org.eclipse.gef 3.11.0.201606061308]

**Comment by Xylandor** [comment]
I also noticed that the tooltips will not show when hovering over the editor tabs when in fullscreen mode either. This tooltip usually displays the full location of the script file that's open. Seems that fullscreen mode breaks tooltip showing.

**Comment by kseniiaguzeeva** [comment]
I cannot reproduce it on MacBook Air. Could you please provide me with information about your screen resolution? Do you use  standalone DBeaver?

**Comment by Xylandor** [comment]
I'm using the latest version fo the Community edition which I installed using the dmg file. No brew install or anything. Everything is set as the default way after installing. I even tried restarting the Mac, but issue persists.

Here's the specs:
MacBook Pro (16-inch, 2019)
Ram: 16GB
Display Res:  3072 x 1920 (This is the default setting)
OS:  MacOS Catalina v10.15.6 (19G2021)

Majority of the time, I connect to two external monitors through a docking station and I usually have two desktops for each monitor. The issue still was existing even when not docked. I did some more playing around and found that once I removed all my extra desktops, I could no longer reproduce the problem. I added back all the desktops thinking that maybe it had something to do with the multiple desktops, but could not replicate it. I'm wondering now if was just some fluke that happened upon installing the first v21 update. I'm good to go now, so I guess this can be closed. Thanks for your help with this.
